js/src/logger.ts

```diff
@@ -1367,6 +1491,19 @@ class BackgroundLogger {
       }
     }
 
+    const attachments = this.attachments;
+    this.attachments = [];
+    // TODO: Error handling.
+    const uploadResponses = await Promise.all(
+      attachments.map((attachment) => attachment.result.get()),
+    );
+    uploadResponses.forEach((result) => {
+      console.log(
```

Comment:
Probably want to remove this before merging?

Reply:
Removed